index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
+</head>
+<body>
+    <header class="header">
+        <a href="#">
+            <img src="imgs/logo.png" alt="YouTube Logo" class="youtube-logo" />
+        </a>
+        <form class="search-bar">
+            <input class="search-input" type="search" placeholder="Search" aria-label="Search" />
+            <button type="submit" class="search-btn">
+            <img src="imgs/magnify.svg" />
+            </button>
+        </form>
+        <div class="menu-icons">
+            <a href="#">
+                <img src="imgs/microphone.svg" alt="Apps" />
+                </a>
+            <a href="#">
+            <img src="imgs/video-plus.svg" alt="Upload Video" />
+            </a>
+            
+            <a href="#">
+            <img src="imgs/bell.svg" alt="Notifications" />
+            </a>
+            <a href="#">
+            <img class="menu-channel-icon" src="http:///unsplash.it/36/36?gravity=center" alt="Your Channel" />
+            </a>
+        </div>
+    </header>
+        
+    <div class="video-section">
+        <div class="main-video">
+            <iframe width="660" height="415" src="https://www.youtube.com/embed/B-ytMSuwbf8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        </div>
+        <div class="side-videos">
+            <div class="side-video">
+                
+                <iframe width="150" height="100" src="https://www.youtube.com/embed/B-ytMSuwbf8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                <div class="video-details">
+                    <a href="#" class="video-title">Video Title</a>
+                    <a href="#" class="video-channel-name">Channel Name</a>
```

Comment:
This is a list of video details, and an <ul> tag with a <li> tag for each video detail would be better appropriate in this situation for semantic reasons. Kindly change the HTML tags.